src/types/json.h

```diff
@@ -211,6 +211,23 @@ struct JsonValue {
     return types;
   }
 
+  StatusOr<std::vector<std::optional<int>>> Toggle(std::string_view path) {
+    std::vector<std::optional<int>> result;
```

Comment:
Seems here we can just use a `std::vector<std::optional<bool>>`?

Reply:
Thanks for pointing out. That is more suitable. Fixed.